test/sparsedir/sparse.jl

```diff
@@ -1319,3 +1319,8 @@ let
     @test issparse(UpperTriangular(full(m))) == false
     @test issparse(LinAlg.UnitUpperTriangular(full(m))) == false
 end
+
+# 16073
+@inferred sprand(1, 1, 1.0)
+@inferred sprand(1, 1, 1.0, rand, Float64)
+@inferred sprand(1, 1, 1.0, x->round(Int,rand(x)*100))
```

Comment:
I just copied this from another test that happened to fail when I did a mistake in fixing this: https://github.com/JuliaLang/julia/blob/e8601e8a19f4801ffd0641d125c50b849f09877a/test/sparsedir/sparse.jl#L592